R/defineModel.R

```diff
@@ -0,0 +1,65 @@
+#' Defining the Model
+#'
+#' Give the complete formula and generate the expected kernel library.
+#'
+#' It processes data based on formula and label_names and creates a kernel
+#' library according to the parameters given in Kern_par.
+#'
+#' @param formula A symbolic description of the model to be fitted.
+#' @param label_names A character string indicating all the interior variables
+#' included in each predictor.
+#' @param data A dataframe to be fitted.
+#' @param Kern_par A dataframe indicating the parameters of base kernels to be
```

Comment:
For parameter names, try to stick to lower case (see http://style.tidyverse.org/syntax.html#object-names). So we should probably change Kern_par to kern_par.

The only exception to this rule is the matrix. By convention we use upper case names for matrices (e.g. X for design matrix, K for kernel matrix etc).